--- build.zig	2021-03-29 16:17:49.839816355 +0000
+++ build.zig2	2021-03-29 16:20:02.663154049 +0000
@@ -7,9 +7,9 @@
     exe.setBuildMode(mode);
 
     // on normal systems this is ok:
-    //exe.linkSystemLibrary("sodium");
+    exe.linkSystemLibrary("sodium");
     // on debian, you have to do this:
-    exe.addObjectFile("/usr/lib/x86_64-linux-gnu/libsodium.a");
+    //exe.addObjectFile("/usr/lib/x86_64-linux-gnu/libsodium.a");
 
     exe.linkLibC();
 
